src/tests/algorithms_test/calorimetry_CalorimeterClusterRecoCoG.cc

```diff
@@ -28,16 +29,19 @@ using eicrecon::CalorimeterClusterRecoCoGConfig;
 using edm4eic::CalorimeterHit;
 
 TEST_CASE( "the calorimeter CoG algorithm runs", "[CalorimeterClusterRecoCoG]" ) {
+  const float EPSILON = 1e-5;
+
   CalorimeterClusterRecoCoG algo("CalorimeterClusterRecoCoG");
 
   std::shared_ptr<spdlog::logger> logger = spdlog::default_logger()->clone("CalorimeterClusterRecoCoG");
   logger->set_level(spdlog::level::trace);
 
   CalorimeterClusterRecoCoGConfig cfg;
   cfg.energyWeight = "log";
-  cfg.sampFrac = 0.0203,
-  cfg.logWeightBaseCoeffs={5.0,0.65,0.31},
-  cfg.logWeightBase_Eref=50*dd4hep::GeV,
+  cfg.sampFrac = 0.0203;
+  cfg.logWeightBaseCoeffs = {5.0,0.65,0.31},
```

Comment:
```suggestion
  cfg.logWeightBaseCoeffs = {5.0,0.65,0.31};
```

Reply:
OT why doesn't clang-tidy catch this? I know it's valid but it is also clearly unintended.